phpunit.xml.dist

```diff
@@ -16,8 +16,18 @@
   <testsuites>
     <testsuite name="Unit Test Suite">
       <directory>*/tests/Unit</directory>
-      <directory>./tests/Unit</directory>
-      <exclude>./dev/tests/Unit/DocFx</exclude>
+      <exclude>dev/tests/Unit/Command/DocFxCommandTest.php</exclude>
+      <exclude>dev/tests/Unit/DocFx/PageTest.php</exclude>
+      <exclude>dev/tests/Unit/DocFx/NodeTest.php</exclude>
+      <exclude>dev/tests/Unit/SplitTest.php</exclude>
+      <exclude>dev/tests/Unit/GitHubTest.php</exclude>
+      <exclude>dev/tests/Unit/ReleaseNotesTest.php</exclude>
+      <exclude>dev/tests/Unit/RunShellTest.php</exclude>
+      <exclude>dev/tests/Unit/JsonFileTest.php</exclude>
+      <exclude>dev/tests/Unit/NewComponentTest.php</exclude>
+      <exclude>dev/tests/Unit/SplitInstallTest.php</exclude>
+      <exclude>dev/tests/Unit/ComponentTest.php</exclude>
+      <exclude>dev/tests/System/ServiceWhitelist/WhitelistTest.php</exclude>
```

Comment:
A system test might not be included automatically due to `<directory>*/tests/Unit</directory>`. So, this line is redundant.